src/app/project/project.component.html

```diff
@@ -51,7 +51,7 @@
                 <ng-container *ngIf="loadingActivity" >
                   <ion-spinner></ion-spinner>
                 </ng-container>
-                <ion-item class="activity-item" *ngFor="let activity of milestone.Activity" color="light">
+                <ion-item class="activity-item" *ngFor="let activity of milestone.Activity" color="light" ng-class="addMargin: 'test'?">
```

Comment:
To change class we using `[ngClass]`. I think we need to change this `ng-class` to `[ngClass]`. And there are no css call `addMargin`